cmd/keycmd/print.go

```diff
@@ -0,0 +1,53 @@
+package keycmd
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/key"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	avago_constants "github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/olekukonko/tablewriter"
+)
+
+func printAddresses(keyPaths []string) error {
```

Comment:
all the other files are one file per command, so we should probably put this in the list file or in a separate package

Reply:
List isn't too long, so I'd suggest putting it there